components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml

```diff
@@ -55,3 +55,5 @@ spec:
         - konflux-otel
         - local-cluster
         - clusters
+        - konflux-qe-team-tenant
```

Comment:
What kind of secrets will these two namespaces retrieve from the Hashicorp Vault?

Reply:
@manish-jangra will retrieve our qe secrets to run tests